src/backend/EdFi.DataManagementService.Backend.Postgresql/Deploy/Scripts/0004_Configure_Replication.sql

```diff
@@ -0,0 +1,9 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+-- Incorrect replication might have been setup by Debezium before this can run.
+-- Delete and recreate correctly with partition support.
+DROP PUBLICATION IF EXISTS to_debezium;
```

Comment:
`to_debezium` is my made-up slot name. Just looked and the default slot name is `debezium`

Reply:
This is all working right now. Can take another look later to tweak if needed.

Reply:
@stephenfuqua 
I guess we are missing to execute  SELECT pg_create_logical_replication_slot('debezium', 'pgoutput'); for creating replication slots.  CC: @msilesgap

Reply:
@msilesgap is adding that to the script file and testing it locally. Thanks

Reply:
I've been having no problem with replication, so I'm not sure why you're adding that?

Reply:
@msilesgap testing confirmed that, the replication slot creation is not required.